### DISCUSSION TITLE
[bug] 无法使用 shift + 点击选中文本

### DISCUSSION THREAD

**Discussion by duanyao** [linuxdeepin/developer-center#2928] [Features Request & Ideas | 特性请求 & 头脑风暴]
deepin-terminal 版本：5.4.0.6
操作：在终端中的文本的任意位置点击鼠标，移动鼠标到文本的另一位置，按下shift的同时再点击鼠标。
期待的结果：选中两次点击之间的文本。
实际的结果：没有任何文本被选中。

这应该是最近的更新（deepin 20 或 20.1）引入的bug，以前的 deepin-terminal 应该没这个问题。

**Comment by wangliang-terminal** [comment]
感谢反馈！！之前在开发deepin 20终端的时候该功能被裁剪了，后期会考虑加上去

**Comment by chenzhiwei** [comment]
为何这类通用的功能非要砍掉呢？

这种选中文本方式几乎适用于所有常用软件，比如 Chrome, Firefox 等等，希望尽快加回来吧。

**Reply by BLumia** [reply]
其实应该算是漏掉的特性，毕竟 v20 终端是从零开始重新开发的 :joy:

**Comment by meiyixiang** [comment]
Now click to select a word, double click to select the entire line of text and you can try it out, I don't know if it helps you
